src/devtools/views/Components/Tree.js

```diff
@@ -310,7 +307,6 @@ export default function Tree(props: Props) {
                   itemCount={numElements}
                   itemData={itemData}
                   itemSize={lineHeight}
-                  overscanCount={3}
```

Comment:
Why? I added this because of visual artifacts on fast scrolling. I don’t think it’s solved yet. Maybe we need to look into why it happens though.

Reply:
Couple of reasons:
1. I don't think overscan actually helps much with artifacts. It only helps with the first paint or two after a fast scroll. After that, react-window is racing to keep up with a passive "scroll" event regardless.
2. With dynamic indentation, I didn't want the size of hidden/offscreen rows visually impacting the indentation size too much, because it felt a bit awkward in some cases.

I don't feel _very_ strongly about this, but it did seem like a positive change.